apps/src/music/player/sequencer/AdvancedSequencer.ts

```diff
@@ -13,20 +14,17 @@ import Sequencer from './Sequencer';
  */
 export default class AdvancedSequencer extends Sequencer {
   private playbackEvents: PlaybackEvent[];
+  private effects: Effects;
 
   constructor(
     private readonly metricsReporter: LabMetricsReporter = Lab2Registry.getInstance().getMetricsReporter()
   ) {
     super();
     this.playbackEvents = [];
+    this.effects = {};
   }
 
-  playSoundAtMeasureById(
-    id: string,
-    measure: number,
-    isBlockInsideWhenRun: boolean,
-    blockId: string
-  ) {
+  playSoundAtMeasureById(id: string, measure: number, blockId: string) {
```

Comment:
This change is unrelated to the PR. The `isBlockInsideWhenRun` parameter is not actually used in `playSoundAtMeasureById`.